src/main/java/com/mcmiddleearth/mcmechat/listener/PlayerListener.java

```diff
@@ -46,7 +47,43 @@ public void onJoin(PlayerJoinEvent e) {
         ChatPlugin chatPlugin = new ChatPlugin();

```

Comment:
Please don't create new plugin instances. If you need to access the plugin instance use static method ChatPlugin.getInstance()